nginx-redirects.conf

```diff
@@ -464,7 +464,7 @@ server {
 server {
     include /etc/nginx/ssl.default.conf;
     server_name planetaryresponsenetwork.org planetaryresponsenetwork.net planetaryresponsenetwork.com www.planetaryresponsenetwork.org www.planetaryresponsenetwork.net www.planetaryresponsenetwork.com;
-    return 301 https://www.zooniverse.org/projects/vrooje/planetary-response-network-and-rescue-global-caribbean-storms-2017;
+    return 301 https://www.zooniverse.org/projects/mrniaboc/prn-hurricane-dorian;
```

Comment:
Should this go to an organization instead?
https://www.zooniverse.org/organizations/zooniverse/planetary-response-network

The org would then include any active projects. That would avoid setting up the redirect each time.